frontend/src/theme/selectStyle.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from "./colors";
+
+const optionBackgroundColor = (isSelected: boolean, isFocused: boolean) => {
+  if (isSelected) {
+    return colors.teal[100];
+  }
+  if (isFocused) {
+    return colors.teal[50];
+  }
+  return colors.white.default;
+};
+
+const selectStyle: any = {
+  singleValue: (provided: any) => ({
+    ...provided,
+    color: colors.gray[500],
+    borderColor: colors.gray[200],
+    boxShadow: "none",
+  }),
+  control: (provided: any, state: any) => ({
+    ...provided,
+    backgroundColor: state.isDisabled ? colors.teal[50] : colors.white.default,
+    borderColor: colors.gray[200],
```

Comment:
When the select is focused, it should update to teal.400. Otherwise, it should default to teal.100:
```suggestion
    borderColor: state.isFocused ? colors.teal[400] : colors.teal[100],
```

one issue with the code above is that after you make your selection, the state will still remain focused until you click elsewhere. I don't think its too much of a big deal, since it technically shows the user their last selection/action

Reply:
the thing is, the only way i could figure out how to remove the ugly blue border when you focus on the non "react-select" library components was to make the border color not change, which is why I made it this way. I'll try again but idk if i'll be able to make it work w the date time pickers